### ISSUE TITLE
SVG import error: "vector is null" in arcTo

### ISSUE THREAD

**Issue by niflostancu** [paperjs/paper.js#1808]
# Description/Steps to reproduce

I'm having a problem importing a SVG file, which was actually generated by webpack's svg-sprite-loader plugin.
I isolated it to a single `<path>` element: (sketch attached, see URL below):
```
<path fill="#c9c8c7"
         d="M-481.302-497.13l-214.207 28.674c-.106-.531-.106-.956-.212-1.38l212.826-37.277a150.441 150.441 0 011.593 9.983"
    />
```
When opened in Inkscape, it shows some kind of triangle (it's way out of the viewport, but it renders).

When trying to import it with Paper, the process fails with:
```
Line 15569: Error: TypeError: vector is null
```
Leading to a `arcTo` function, near lines:
```
matrix = new Matrix().translate(center).rotate(rotation).scale(rx, ry);
vector = matrix._inverseTransform(from);
extent = vector.getDirectedAngle(matrix._inverseTransform(to));
```

# Link to reproduction test-case

http://sketch.paperjs.org/#V/0.12.4/S/jVPJbtswEP0VgrnYQLhKshY4ySVoe2hOcXqpCkShaFsxRQoiY7so8u8ZO1ZjFS1QARLJN+8NR7P8wrZqNS7w/UYHtcaXWLn6cO6qTve0692zVoE2bef6cP/t8+RxfrNvDdrq3jfOXpVYUF5ipC3oGrsC4GHxiWQA+VDZujLOagCtK/HNdWnnfrsqLUIInFhf1Aps6xC6grHupTfU9StWK6aNbrUNngkqWInPFOpMoXpdhWarlWtbZ/1RbP3FiN/Xyw/Bbreju+jIE3meMy6ZlAQoxP+0odqTP9UQ7V/VknPODr9yRv5vYrE3jd38O6qjeRwG5LaD90MzINS7l17pJYg1tTqw28XtbyPhtA71yBN49gpKO759QN8zCA3hu0ppzwb85GI4FqPqo0nMo6SKl09xfokkl5zwhHAxPanOyaLE10dw3lVhjZaNMYBfqFxlKj3x358a8DsSZ4JGXJI4T6mIDJEippKnSGZ0lsaKUMFnhCaROO3yBD5SSAKyzMCGZnJGopTKNK1EwmkcCzSsHHEhaJJHKKd5Fg3Xs0ObHmoG6+O0tCW0tvXOaGrcajKeC70f5mIKSbS6/7K4+woamKMnaM5N5xroYlx8//H6Bg==

# Expected result

It shouldn't crash with a JS error, worst case, it should refuse to render that item.

**Comment by sasensi** [comment]
The issue comes from the fact that the SVG path is invalid, if you look at the [spec](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Paths), you will see that the `a` command expects 7 arguments and only get 5 in your example.

As how to handle invalid SVG, I am not sure, @lehni, what do you think, should we let the error be thrown or catch it ?

**Comment by niflostancu** [comment]
Yep, I figured it. The thing is, Inkscape didn't complain at all about such invalid definitions.
I removed them and now it's okay.
Graceful error handling (optionally, emitting warning output) is desired when accepting unknown input, though.
Thanks!

**Comment by lehni** [comment]
@niflostancu could you post a screenshot of what Inkscape imports?
